hack/ginkgo-e2e.sh

```diff
@@ -89,7 +89,7 @@ fi
 export PATH=$(dirname "${e2e_test}"):"${PATH}"
 "${ginkgo}" "${ginkgo_args[@]:+${ginkgo_args[@]}}" "${e2e_test}" -- \
   "${auth_config[@]:+${auth_config[@]}}" \
-  --host="https://${KUBE_MASTER_IP-}" \
+  --host="https://${KUBE_MASTER_IP:-}" \
```

Comment:
I think as it's currently written, there's no real difference between `:-` and `-` - it will either give you the value of `$KUBE_MASTER_IP` or the empty string.